### ISSUE TITLE
Write doc page for model-8/model-10 issue

### ISSUE THREAD

**Issue by bizzinho** [SPL-ethz/snow#34]
Convert from latex (https://www.overleaf.com/project/61c4470e0f950bebf892149d) to rst/sphinx doc.

**Comment by ltdeck** [comment]
Do you think it is better to have a rst doc compared to the compiled pdf from overleaf?

**Comment by bizzinho** [comment]
you can use pandoc to generate a markdown from tex, I would recommend that

**Comment by bizzinho** [comment]
Probably, there are also online converters, maybe look into that? If you give me something as .md or .rst I will incorporate it into the doc

**Comment by ltdeck** [comment]
Thanks. I will look into pandoc

**Comment by bizzinho** [comment]
I prepared the switch in the code in 9dbc262 @ltdeck . Please fill it in.

It's ugly, but doing this nicer would require more time. In a future state, maybe we can group all simulation options (including dt) into a separate object. But that would be a backwards-compatibility-breaking change, so shouldn't do it lightly.

**Comment by bizzinho** [comment]
hey @ltdeck , where is this file? :)

**Comment by ltdeck** [comment]
@bizzinho I combined it with the overall validation file. Please have a look, then I will convert it.

**Comment by bizzinho** [comment]
I have converted it and am updating some other minor stuff around the docs. Will build some proper CD if I have the time.

**Comment by bizzinho** [comment]
OK, a bit of a pain to do this right with mathjax. I cannot figure out how to make the equation labels for aligned equation blocks properly right-aligned, but it's not worth the additional time and effort.

I noticed there was a missing bracket in equation 11 and those derived from it (i.e., equation 1 in the new numbering). Please have a look at the docs by pulling the 3D_docs branch and opening `docs/index.html`. Let me know whether this is ok for you, then I will merge.
